cmd/cmd.go

```diff
@@ -79,6 +79,7 @@ func NewPackCommand(logger ConfigurableLogger) (*cobra.Command, error) {
 	rootCmd.AddCommand(commands.InspectImage(logger, imagewriter.NewFactory(), cfg, packClient))
 	rootCmd.AddCommand(commands.NewStackCommand(logger))
 	rootCmd.AddCommand(commands.Rebase(logger, cfg, packClient))
+	rootCmd.AddCommand(commands.DownloadSBOM(logger, packClient))
```

Comment:
Curious... what are we doing about `pack inspect <image> --bom`?

Reply:
As of today, it's technically deprecated. But it might be a little heavy-handed to remove the functionality entirely; if the user wants to look at older images. Perhaps a warning/deprecation message is in order?

Reply:
_Addressed in latest commit._

I'm displaying the following warning message:
> Warning: Using the '--bom' flag with 'pack inspect-image <image-name>' is deprecated. Users are encouraged to use 'pack download sbom <image-name>'.